src/tests/ftest/io/nvme_io.yaml

```diff
@@ -2,7 +2,7 @@
 # required quantity is indicated by the placeholders
 
 hosts:
- servers:
+ test_machines:
```

Comment:
You can take advantage of the newer TestWithServers class and get self.hostlist_servers and self.hostlist_clients defined if this is slightly restructured into:

test_machines:
  test_servers:
    - boro-A
    - boro-B
    - boro-C
    - boro-D
  test_clients:
    - boro-E